frontend/src/chrome/TextLink.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
Any reason not to rename this to just `Link`? I think it's a good thing that it collides with `react-router-dom`'s `Link`, because they do the same thing.

Reply:
There are a lot of places where we are using `Link` which should use this instead.  I'll open an issue to audit the use of `Link` and use `TextLink` instead where we can.

Reply:
https://github.com/qri-io/qrimatic/issues/197